constraints.txt

```diff
@@ -110,27 +113,28 @@ jupyter_core==5.3.1
 jupyter_server==2.7.0
 jupyter_server_terminals==0.4.4
 jupyterlab-pygments==0.2.2
-jupyterlab-widgets==3.0.7
+jupyterlab-widgets==3.0.8
 keyring==24.2.0
 kgcl-rdflib==0.5.0
 kgcl-schema==0.5.0
-kgx==2.0.2
+kgx==1.7.2
```

Comment:
😱

Reply:
Can I help here?  What in KGX is causing the dependency issues?

Reply:
Yes please!
```
Because kgx (2.2.1) depends on networkx (>=2.8,<3.0)
 and no versions of kgx match >2.2.1,<3.0.0, kgx (>=2.2.1,<3.0.0) requires networkx (>=2.8,<3.0).
So, because test depends on both networkx (^3.1) and kgx (^2.2.1), version solving failed.
```
This is one for beginners. I'm trying to find others.